www/include/monitoring/recurrentDowntime/formDowntime.html

```diff
@@ -183,7 +183,7 @@
 </script>
         {/literal}
 
-<div id="msg_err" style="color: red; text-align: center; font-weight: bold; margin: 4px; {if !$msg_err }display: none;{/if}">{$msg_err}</div>
+<div id="msg_err" style="color: red; text-align: center; font-weight: bold; margin: 4px;display: none;"></div>
```

Comment:
if there is a form validation error, is it still display ?

Reply:
This div is used on code later on, with a message + show

Reply:
I've been testing out all possible messages and works like a charm